Makefile

```diff
@@ -344,6 +345,18 @@ uninstall:
 	@echo "$(WHALE) $@"
 	@rm -f $(addprefix $(DESTDIR)/bin/,$(notdir $(BINARIES)))
 
+ifeq ($(GOOS),windows)
+install-deps:
+	script/setup/install-runc
```

Comment:
runc on windows?

Reply:
you should try it! it's amazing 😂

Reply:
added a TODO.. don't have a windows development environment ATM.. :-)